vite.config.ts

```diff
@@ -0,0 +1,8 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), tsconfigPaths()],
```

Comment:
This appears to run only esbuild and not tsc or eslint; would it be possible/feasible to get TypeScript/ESLint checks run on each update? Or perhaps the performance hit is not worth it (in which case #89 is a wontfix).

Reply:
I think there's a way of checking for type errors? It might it can be done with [vite-plugin-checker](https://vite-plugin-checker.netlify.app/).

Reply:
Added that plugin, seems to be what you're wanting and the performance isn't really taking a hit imo

Reply:
Great; thanks. Does the plugin also support running ESLint?

Reply:
It does, details are [here](https://vite-plugin-checker.netlify.app/). I'll add that right now o7

Reply:
Added! I'll resolve this comment for now